NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs

```diff
@@ -296,7 +321,14 @@ private AddDownloadDialog(Gtk.Builder builder, AddDownloadDialogController contr
                 quality = Quality.Resolution;
                 resolution = _mediaUrlInfo.VideoResolutions[(int)_qualityRow.GetSelected()];
             }
-            _controller.PopulateDownloads(_mediaUrlInfo!, fileType, quality, resolution, (Subtitle)_subtitleRow.GetSelected(), _saveFolderString, _speedLimitSwitch.GetActive(), _cropThumbnailSwitch.GetActive(), _usernameRow.GetText(), _passwordRow.GetText());
+            if(_keyringRow.GetSelected() == 0 || _keyringRow.GetSelected() == 4294967295 || !_authRow.GetEnableExpansion())
```

Comment:
Maybe `uint.MaxValue` or at least a comment would be better than just magic number?

Reply:
The magic number is just https://docs.gtk.org/gtk4/const.INVALID_LIST_POSITION.html

Probably best to just add a comment, could you do it?

Reply:
Ok, fixed with https://github.com/NickvisionApps/TubeConverter/pull/330/commits/9acc846e896db454635d0876c395f9a56c2f45b1